zebrad/src/components/inbound.rs

```diff
@@ -28,8 +32,13 @@ use downloads::Downloads;
 
 type Outbound = Buffer<BoxService<zn::Request, zn::Response, zn::BoxError>, zn::Request>;
 type State = Buffer<BoxService<zs::Request, zs::Response, zs::BoxError>, zs::Request>;
-type Verifier = Buffer<BoxService<Arc<Block>, block::Hash, VerifyChainError>, Arc<Block>>;
-type InboundDownloads = Downloads<Timeout<Outbound>, Timeout<Verifier>, State>;
+type BlockVerifier = Buffer<BoxService<Arc<Block>, block::Hash, VerifyChainError>, Arc<Block>>;
+type TxVerifier = Buffer<
+    BoxService<transaction::Request, transaction::Response, TransactionError>,
+    transaction::Request,
+>;
+type InboundDownloads = Downloads<Timeout<Outbound>, Timeout<BlockVerifier>, State>;
```

Comment:
Name nitpick, but it might help avoid confusion:
* should we import `downloads::Downloads` as `BlockDownloads`?
* should we rename `InboundDownloads` to `InboundBlockDownloads`?

Reply:
That's much better, I missed those. Changed in 6969c3109